ui/tests/acceptance/mfa-method-test.js

```diff
@@ -263,6 +263,11 @@ module('Acceptance | mfa-method', function (hooks) {
           .dom('[data-test-ttl-value="Period"]')
           .hasValue(model.period.toString(), 'Period form field is populated with model value');
         assert.dom('[data-test-select="ttl-unit"]').hasValue('s', 'Correct time unit is shown for period');
+      } else if (key === 'algorithm' || key === 'digits' || key === 'skew') {
```

Comment:
I had to fix the acceptance test in backport PR, for somewhat reason it didn't fail in the main branch.
I will also push a separate PR just to main for fixing the the same tests.